### ISSUE TITLE
Would you fix errata in comment in stock.js?

### ISSUE THREAD

**Issue by devginie** [dc-js/dc.js#1131]
Before : 
```
//```javascript
//d3.csv('data.csv', function(data) {...};
//d3.json('data.json', function(data) {...};
//jQuery.getJson('data.json', function(data){...});
//```
```

After :
```
//```javascript
//d3.csv('data.csv', function(data) {...});
//d3.json('data.json', function(data) {...});
//jQuery.getJson('data.json', function(data){...});
//```
```

**Comment by gordonwoodhull** [comment]
Thanks for the report! Will do.

**Comment by gordonwoodhull** [comment]
Will be fixed in 2.0 beta 28
